query_optimizer/tests/execution_generator/Select.test

```diff
@@ -656,7 +656,7 @@ SELECT SUM(EXTRACT(SECOND FROM value))
 FROM dates;
 --
 +-------------------------------+
-|SUM(EXTRACT(SECOND FROM value))|
+|SUM(EXTRACT(second FROM value))|
```

Comment:
I was wondering if there are some standards about whether we should capitalize all the units.